doc/autobuild.py

```diff
@@ -64,6 +53,19 @@
 from kivy.factory import Factory
 from kivy.lib import osc, ddsfile, mtdev
 
+
+ignore_list = (
```

Comment:
this change should be probably ok, it's ignore only for autodoc, not for kivy directly.